kola/tests/systemd/sysext.go

```diff
@@ -50,3 +66,24 @@ func checkSysextSimple(c cluster.TestCluster) {
 	// Second check after reloading the extensions (e.g., to add/remove/update them)
 	checkHelper(c)
 }
+
+func checkSysextCustomDocker(c cluster.TestCluster) {
+	cmdNotWorking := `if docker run --rm ghcr.io/kinvolk/busybox true; then exit 1; fi`
+	cmdWorking := `docker run --rm ghcr.io/kinvolk/busybox echo Hello World`
+	// First assert that Docker doesn't work because Torcx is disabled
+	_ = c.MustSSH(c.Machines()[0], cmdNotWorking)
+	// We build a custom sysext image locally because we don't host them somewhere yet
+	_ = c.MustSSH(c.Machines()[0], `git clone https://github.com/flatcar/sysext-bakery.git && git -C sysext-bakery checkout 2d6ab6bdba96476138c7db841e9a4c4ed38bd3fc`)
+	// Flatcar has no mksquashfs and btrfs is missing a bugfix but at least ext4 works
+	// The first test is for a fixed Docker version, which with the time will get old and older but is still expected to work because users may also "freeze" their Docker version this way
+	_ = c.MustSSH(c.Machines()[0], `ONLY_DOCKER=1 FORMAT=ext4 sysext-bakery/create_docker_sysext.sh 20.10.21 docker && ONLY_CONTAINERD=1 FORMAT=ext4 sysext-bakery/create_docker_sysext.sh 20.10.21 containerd && sudo mv docker.raw containerd.raw /etc/extensions/`)
+	_ = c.MustSSH(c.Machines()[0], `sudo systemctl restart systemd-sysext`)
+	// We should now be able to use Docker
+	_ = c.MustSSH(c.Machines()[0], cmdWorking)
+	// The next test is with a recent Docker version, here the one from the Flatcar image to couple it to something that doesn't change under our feet
+	version := string(c.MustSSH(c.Machines()[0], `bzcat /usr/share/licenses/licenses.json.bz2 | grep -m 1 -o 'app-emulation/docker[^:]*' | cut -d - -f 3`))
+	_ = c.MustSSH(c.Machines()[0], `ONLY_DOCKER=1 FORMAT=ext4 sysext-bakery/create_docker_sysext.sh `+version+` docker && ONLY_CONTAINERD=1 FORMAT=ext4 sysext-bakery/create_docker_sysext.sh `+version+` containerd && sudo mv docker.raw containerd.raw /etc/extensions/`)
```

Comment:
I think we can set the [`ARCH`](https://github.com/flatcar/sysext-bakery/blob/main/create_docker_sysext.sh#L4) argument. Otherwise it fails on arm64:
```
cluster.go:117: bash: line 1: /usr/bin/docker: cannot execute binary file: Exec format error
```